src/data/local/github/tracks-node/track-6/steps/step-3.md

```diff
@@ -1,8 +1,18 @@
-# Step 3/3 - Tests
-### Estimated time: 3 hours
+# Step 3/5 - The token
+### Estimated time: 2 hours
 
-Now it's time to write some tests for the new mutation. it should be easier now that you have everything setup. Remember to test every single scenario you have in mind.
+Now you're going to finish your `login` mutation returning a proper token instead of an empty string. Take a look at an example of the token you're going to implement:
 
-Just to remember another important thing to test that didn't happen on the previous track (login): the database changes. You should not only test if the return of the mutation is the expected, but also if the database was updated as it should be. For example, if you have 4 users on the database, and you add one more, you should check on your test if there are 5 users after the mutation is called, as well as if all the new user info were properly inserted.
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+```
 
-After you write all tests and they are running properly, you can open a PR and go to next track.
+This may look like a random string, but this token has a lot of power, because there are some information encoded on it. The server can know only with a token, for example:
```

Comment:
```suggestion
This may look like a random string, but this token has a lot of power, because there are some information encoded in it. The server can know only with a token, for example:
```
eu acho 😅